src/Features/OnboardingTasks/Tasks/Purchase.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\Onboarding;
+use Automattic\WooCommerce\Admin\PluginsHelper;
+
+/**
+ * Purchase Task
+ */
+class Purchase {
+	/**
+	 * Get the task arguments.
+	 *
+	 * @return array
+	 */
+	public static function get_task() {
+		$products = self::get_products();
+
+		return array(
+			'id'             => 'purchase',
+			'title'          => count( $products['remaining'] ) === 1
+				? sprintf(
+					/* translators: %1$s: list of product names comma separated, %2%s the last product name */
+					__(
+						'Add %s to my store',
+						'woocommerce-admin'
+					),
+					$products['remaining'][0]
+				)
+				: __(
+					'Add paid extensions to my store',
+					'woocommerce-admin'
+				),
+			'content'        => count( $products['remaining'] ) === 1
+				? $products['purchaseable'][0]['description']
+				: sprintf(
+					/* translators: %1$s: list of product names comma separated, %2%s the last product name */
+					__(
+						'Good choice! You chose to add %1$s and %2$s to your store.',
+						'woocommerce-admin'
+					),
+					implode( ', ', array_slice( $products['remaining'], 0, -1 ) ) . ( count( $products['remaining'] ) > 2 ? ',' : '' ),
+					end( $products['remaining'] )
+				),
+			'action_label'   => __( 'Purchase & install now', 'woocommerce-admin' ),
+			'action_url'     => '/setup-wizard',
```

Comment:
I believe this task doesn't have the `action_url`

Reply:
Ah, I thought this was removed by Joel in https://github.com/woocommerce/woocommerce-admin/pull/7616, but I guess I was mistaken.  When rebasing, I dropped any changes inside `get_all()` - which appear to just be new lines, but cc @joelclimbsthings just in case.

Reply:
Confirmed, just new lines in that file. Didn't touch the `action_url` here.